README.md

```diff
@@ -310,6 +310,12 @@ added to the `CMAKE_PREFIX_PATH` when building SYCL-BLAS (see
 
 ## Setup
 
+**IMPORTANT NOTE:** The `TARGET` CMake variable is no more supported, it has
+been replaced by `TUNING_TARGET`, which accepts the same options.
+`TUNING_TARGET` effects only the tuning configuration, no more the target
```

Comment:
```suggestion
`TUNING_TARGET` affects only the tuning configuration and has no effect on the target
```

Reply:
I got confused with the wording here. I think this should make it a bit more clear.

Reply:
Sure, thanks @fabiomestre!